src/index.js

```diff
@@ -617,8 +625,26 @@ import {
 
             // Any new message that does not reply-to or mention any user (1)
             else if (eventType === ChatEventType.MESSAGE_POSTED && !targetUserId) {
+
+                /** @type {[m:(c:string) => boolean, b:(c:BotConfig, e:Election, t:string) => string][]} */
+                const rules = [
+                    [isAskedForCurrentPositions, sayNumberOfPositions]
+                ];
+
+                const matched = rules.find(([expr]) => expr(content));
```

Comment:
The guards are currently in this order so the more specific ones match first. We need to improve and test those regex guards if we want to go this way, which we don't have time to do before the SO election.

Reply:
Definitely - I do not intend to add anything else to the guard map until the SO election passes - just created the scaffolding and the first guard that is covered with tests. Each guard moved in this map would need to be tested for compatibility. Ideally, I would like us to get a set of guards that uniquely identify commands